_saas-integrations/zoom/v1/zoom-v1.md

```diff
@@ -29,12 +29,109 @@ name: "zoom"
 display_name: "Zoom"
 
 singer: true
-status-url: ""
+status-url: "https://status.zoom.us/"
 
 tap-name: "Zoom"
 repo-url: https://github.com/singer-io/tap-zoom
 
 this-version: "1"
+
+api: |
+  [Zoom REST API v2](https://marketplace.zoom.us/docs/api-reference/introduction){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: false
+
+api-type: "platform.zoom"
+
+#historical: ""
+frequency: "1 hour"
+tier: "Free"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: |
+      **{{integration.display_name }} authentication apps**. You'll need one or the other to connect your {{integration.display_name }} account to Stitch:
```

Comment:
This is super nitpicky, but I still think it's worth pointing out. When writing items in here, consider how the first sentence in this entire section (**[integration] Setup requirements**) should make items in this list sound. Ex:

```
## Zoom setup requirements

To set up Zoom in Stitch, you need:

   - **Zoom authentication apps**.
```

While `Zoom authentication apps` is technically correct for what you're talking about here (which are **apps**), the user really only needs a single authentication app, right? To that end, I would change `Zoom authentication apps` to be more specific and read a bit smoother: `A Zoom authentication app`